Frameworks/Foundation/NSFileManager.mm

```diff
@@ -677,15 +746,20 @@ - (BOOL)isWritableFileAtPath:(id)pathAddr {
 }
 
 /**
- @Status Stub
+ @Status Interoperable
 */
 - (BOOL)isExecutableFileAtPath:(NSString*)path {
-    UNIMPLEMENTED();
+    struct stat st;
+    if (EbrStat([path UTF8String], &st) == 0) {
```

Comment:
this is actually not testing that.

you need to use `access` -- you're telling whether the path is marked executable for _anyone_, but the API is asking whether it's executable for _us_